### ISSUE TITLE
Include custom CSS file in site generation

### ISSUE THREAD

**Issue by danielk333** [plandes/zotsite#22]
New feature:
Would be useful to be able to include a custom css file with the static site generation for styling according to e.g. institute standards or dark-mode.

**Comment by plandes** [comment]
Yes, this would be very trivial to implement.  Right now, it gets all static content from a Python resource directory and just copies everything over in to the target website directory structure.  You could add a command line option, or perhaps better, a configuration to copy over custom CSS styles.

**Comment by plandes** [comment]
@danielk333 At the moment I do not have time to implement this. If you would like to, please do and submit a pull request.  Thank you.

**Comment by plandes** [comment]
@danielk333 I decided to see what was involved and got something to work for dark mode using a switch in the header.  However, I didn't get very far as it doesn't apply to all elements (i.e. left nav's font color doesn't update).  I added this to the [dark mode branch](https://github.com/plandes/zotsite/tree/dark-mode) in case you wanted to pick up where I left off.  It would be great to be able to add these customizations so let me know if you're willing to finish it.
